### PR TITLE
Recent HPX master has changed some namespaces for the executor traits specialization

### PR DESCRIPTION
**Pull request by hkaiser**

### CODE REVIEW

File: src/hpx/kokkos/config.hpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+///////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <hpx/config/version.hpp>
+
+#if HPX_VERSION_FULL >= 0x011100
```
Comment (msimberg) [inline src/hpx/kokkos/config.hpp:12]: Isn't this >= 1.17.0, since it's a hexadecimal literal?
```suggestion
#if HPX_VERSION_FULL >= 0x010B00
```

**Review by msimberg — CHANGES_REQUESTED** [review]
Thanks for the fixes @hkaiser! Not sure about the version check, otherwise this looks good.

I suppose this warrants a new patch release. I can take care of that separately if you'd like.

File: src/hpx/kokkos/config.hpp (hunk 1 above)
Comment (hkaiser) [reply src/hpx/kokkos/config.hpp:12]: I think it's correct the way I proposed it, see https://github.com/STEllAR-GROUP/hpx/blob/f711e4192411cc049889ba85614f211d9e86538b/libs/core/config/cmake/templates/config_version.hpp.in#L28-L35

Also, yes, please do a patch release as you see fit.

Comment (msimberg) [reply src/hpx/kokkos/config.hpp:12]: Oh... yes, that's interesting. And you're right, sorry for the noise.

**Review by msimberg — APPROVED** [review]

**Comment by msimberg** [comment]
@hkaiser do you want someone (@G-071) to test this as well before merging, or ok to merge as-is (at least the github actions workflow passed)? You have rights to merge as well of course, feel free to do so.

**Comment by hkaiser** [comment]
> @hkaiser do you want someone (@G-071) to test this as well before merging, or ok to merge as-is (at least the github actions workflow passed)? You have rights to merge as well of course, feel free to do so.

Let's have @diehlpk test it as he ran across the issue before merging.

**Comment by G-071** [comment]
The test seem to fail in the install dependencies step now though. Seems to be unrelated to hpx-kokkos itself.

From a quick glance, the test tries to build Kokkos develop together with a rather old HPX 1.9.0. 
@msimberg Should we fix both the HPX and Kokkos versions to the latest stable releases instead of using Kokkos develop?

**Comment by msimberg** [comment]
Thanks @G-071!

> The test seem to fail in the install dependencies step now though. Seems to be unrelated to hpx-kokkos itself.

> From a quick glance, the test tries to build Kokkos develop together with a rather old HPX 1.9.0. which might cause the issue?
> @msimberg Should we fix both the HPX and Kokkos versions to the latest stable releases instead of using Kokkos develop?

Yeah, for sure... I assume something just broke on Kokkos develop. HPX 1.9.0 _should_ not be causing that. Let me try to quickly put together a PR that updates the CI workflow. It might be a good idea to test a few versions of HPX and Kokkos.

**Comment by msimberg** [comment]
See https://github.com/STEllAR-GROUP/hpx-kokkos/pull/26.

**Comment by msimberg** [comment]
CI is pretty comprehensive regarding different version combinations right now, so I've merged this. CUDA is not tested however...

I'm going to start preparing a new release. @G-071 is there any further testing you'd like to do before I go ahead with a release?

**Comment by G-071** [comment]
@msimberg No, it looks all good from my end. With the current hpx-kokkos master and hpx master, Octo-Tiger builds just fine and the tests are passing. I think we are good for a new release!

**Pull request merged**